ocs_ci/ocs/amq.py

```diff
@@ -446,7 +447,170 @@ def run_in_bg(self, namespace=constants.AMQ_NAMESPACE, value='10000', since_time
 
         return threads
 
-    # ToDo: Install helm and get kafka metrics
+    def run_amq_benchmark(
```

Comment:
Please apply these changes to an existing test case and verify the PR

Reply:
done